### DISCUSSION TITLE
Amis community outside of China

### DISCUSSION THREAD

**Discussion by abasse** [baidu/amis#5456] [General]
Hi all, it would be fantastic to have a community for English speaking amis contributers outside of QQ and 如流群 as we don't have access to this services outside of China.

Any ideas and suggestions are welcome.

**Comment by h7ml** [comment]
Can also be discussions (https://github.com/baidu/amis/discussions) and [issues] (https://github.com/baidu/amis/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) is discussed

**Reply by abasse** [reply]
Thanks for taking the time to answer this. I know we can use GitHub for this but a chat based system eg. discord would be a more vibrant solution.
